ext/storage_sources/s3_store/s3_log_system.h

```diff
@@ -27,6 +27,8 @@ class S3LogSystem : public Aws::Utils::Logging::LogSystemInterface {
     void LogStream(Aws::Utils::Logging::LogLevel logLevel, const char *tag,
       const Aws::OStringStream &messageStream) override;
     void LogVerboseMessage(int32_t verbosityLevel, const std::string &message);
```

Comment:
`LogVerboseMessage` can be defined in the `private` section since we don't use it anywhere outside of the class. We should only expose the wrappers.

I think we could append the `const` keyword to `LogVerboseMessage`, `LogVerboseErrorMessage` and `LogVerboseDebugMessage`, please could you check and add it if possible?